portfolio/src/main/webapp/index.html

```diff
@@ -33,27 +40,32 @@ <h2>A few bullet points about me</h2>
           </ul> 
         </div> 
       
-        <div id="projects">
+        <div id="projects" class="navbarSection">
           <h2>What I have been working on</h2> 
           <h3>(Click on each project to see more details!)</h3> 
-          <div id="project-wrapper">
-            <div id="project-nlp" onClick="addProjectDescription('nlp')"> 
+          <div id="projectWrapper">
+            <div id="projectNlp" onClick="addProjectDescription('Nlp')"> 
```

Comment:
I'd suggest that you just pass the uppercase strings, like 'NLP'

Reply:
Done.